pool/pool_aio_test.go

```diff
@@ -31,14 +31,33 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/user"
 	"github.com/nspcc-dev/neofs-sdk-go/waiter"
 	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
 )
 
 var (
 	defaultTimeOut = 5 * time.Second
 	tickInterval   = 1 * time.Second
+
+	tickEpoch = []string{
+		"neo-go", "contract", "invokefunction", "--wallet-config", "/config/node-config.yaml",
+		"-a", "NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP", "--force", "-r", "http://localhost:30333",
```

Comment:
some magic addresses to me in that repo, but mb that is ok for the tests

Reply:
When we fix AIO start problem, this command will be removed from tests

Reply:
ok but the [same question](https://github.com/nspcc-dev/neofs-sdk-go/pull/467#discussion_r1273424492)